frontend/src/components/utils/Term.tsx

```diff
@@ -59,6 +59,8 @@ export default class extends React.Component<Props, State> {
               margin-left: 5px;
               width: 16px;
               cursor: pointer;
+              postion: relative;
+              z-index: 10;
```

Comment:
Hmm, what was the issue with this? I believe, `stopPropagation` is indeed the fix, but these two are not clear to me yet.

Reply:
will remove this, this is what I try to see whether css can help.